plugin/src/it/basic-lowercase-dockerfile/verify.groovy

```diff
@@ -19,3 +19,6 @@
  */
 File imageIdFile = new File(basedir, "target/docker/image-id")
 assert imageIdFile.isFile()
+
+File imageDigestFile = new File(basedir, "target/docker/image-digest")
+assert imageDigestFile.isFile()
```

Comment:
Would prefer if you create a new test that only tests this functionality, all the tests don't need to test it as well.